React/tic-tac/src/app/components/Router/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { Fragment } from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import Game from '@screens/Game';
+
+import Login from '../Login';
```

Comment:
Is `Login` a component or a screen? Move it to the appropriate folder and reference it here using an absolute import. 😺